tests/evaluate/test_default_evaluator.py

```diff
@@ -2834,6 +2859,46 @@ def test_eval_df(predictions, targets, metrics, inputs, truth, context):
     ]
 
 
+def test_evaluate_no_model_or_predictions_specified():
```

Comment:
Verify that the default evaluator still requires predictions or model to be specified